app/models/form526_submission.rb

```diff
@@ -157,12 +157,29 @@ class Form526Submission < ApplicationRecord
   belongs_to :user_account, dependent: nil, optional: true
 
   validates(:auth_headers_json, presence: true)
+  enum backup_submitted_claim_status: { accepted: 0, rejected: 1 }
   enum submit_endpoint: { evss: 0, claims_api: 1, benefits_intake_api: 2 }
 
   scope :pending_backup_submissions, lambda {
     where(aasm_state: 'delivered_to_backup')
       .where.not(backup_submitted_claim_id: nil)
   }
+  scope :in_process, lambda {
+    where(submitted_claim_id: nil, backup_submitted_claim_status: nil)
+      .where('created_at >= ?', 1.day.ago)
```

Comment:
this should probably be more than a day, to account for long running back up submissions. Also we should make it a constant

Reply:
Increased to 7 days and made constant in [ccd6694](https://github.com/department-of-veterans-affairs/vets-api/pull/17274/commits/ccd66940f6df82e0f004b5fdce9c60a6bda9be70).